pimux/function.py

```diff
@@ -281,5 +286,33 @@ def volumeControl(self,stream="ring",volume=5):
 
 
 
+class notification:
+    '''
+    The class notification has two methods:
+    - notification
+    - removeNotification
+    '''
+    def __init__(self):
+        pass
+    def notification(self,title="title",content="content", *args, **kwargs):
+        '''
+        This method creates a notification. 'title' and
+        'content' are mandatory, and all other flags in
+        termux-notification are optional arguments
+        '''
+        cargs=[]
+        for i in args:
+            cargs.append(f"--{i}")
+        for i in kwargs:
+            cargs.append(f"-{i} {kwargs[i]}" if len(i) == 1 else f"--{i} {kwargs[i]}") 
+        self.value=t.compute(f"temux-notification -t {title} -c {content} {' '.join(cargs)}")
```

Comment:
what if we cld have method for each flag

Reply:
> what if we cld have method for each flag
maybe, but i have no idea how to do that. how would it tie in to the final command? do you mean a method to set each flag?

Reply:
yea, infact method only those which cld really be useful. If the user of this program can really graps the metbods that are useful it would be better. what do you say . Its the same way as you have created for notification method above. Further on making changes first clone the latest version because it dont make work tedious and always put your new class at the bottom of the fuction.py If you encounter any problem you can share.

Reply:
sure, that should work